test/RestApi.test.ts

```diff
@@ -21,5 +27,32 @@ describe('REST API Test', () => {
             }
         });
     });
-}); 
+    
+    test('Validate CodeDeploy Deployment Config', () => {
+       assert.hasResourceProperties('AWS::CodeDeploy::DeploymentGroup', {
+           DeploymentConfigName: 'CodeDeployDefault.LambdaLinear10PercentEvery1Minute'
+       }); 
+    });
+    
+    test('Validate Lambda function Runtime', () => {
```

Comment:
Same here about the title of the test - just so I can look at the name of a test in the list of failed stuff and know exactly what the issue is